.drone.starlark

```diff
@@ -349,6 +349,43 @@ config = {
 			],
 			'runCoreTests': True,
 		},
+		'core-api-acceptance-latest-nightly': {
+			'suites': [
+				'apiAuth',
+				'apiAuthOcs',
+				'apiCapabilities',
+				'apiComments',
+				'apiFavorites',
+				'apiMain',
+				'apiSharees',
+				'apiShareManagement',
+				'apiShareManagementBasic',
+				'apiShareOperations',
+				'apiShareReshare',
+				'apiShareUpdate',
+				'apiSharingNotifications',
+				'apiTags',
+				'apiTrashbin',
+				'apiVersions',
+				'apiWebdavLocks',
+				'apiWebdavLocks2',
+				'apiWebdavMove',
+				'apiWebdavOperations',
+				'apiWebdavProperties',
+				'apiWebdavUpload',
+			],
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
```

Comment:
compared to the section above, this new section runs against `latest` rather than `daily-master-qa`